plugins/source/aws/client/testing.go

```diff
@@ -51,5 +51,46 @@ func AwsMockTestHelper(t *testing.T, table *schema.Table, builder func(*testing.
 		Version:      version,
 		Tables:       []string{table.Name},
 		Destinations: []string{"mock-destination"},
-	})
+	}, source.WithTestPluginAdditionalValidators(validateTagStructure))
+}
+func extractTables(tables schema.Tables) []*schema.Table {
```

Comment:
```suggestion
func extractTables(tables schema.Tables) schema.Tables {
```
Maybe? Should be the same

Reply:
There are 100 other occurrences of us using `[]*schema.Table` rather than `schema.Tables` if that is a change we want to make we should consider how to enforce it and change it everywhere

Reply:
I think @disq is just saying that this particular signature is inconsistent: it takes `schema.Tables` as an argument but then returns `[]*schema.Table`, which is actually the same thing - we may as well keep it consistent